### ISSUE TITLE
Centralize information about how to make use of Tempo for distributed tracing

### ISSUE THREAD

**Issue by zalegrala** [grafana/grafana#73709]
**Why is this needed**:

Community questions are becoming more frequent about how to make use of Tempo in their distributed tracing system, and
specifically the integrations that exist within Grafana for visualizing and troubleshooting a system under traces instrumentation.  It would be nice to have a single place to point folks at where they could get an understanding of the current state of the integrations that we support.  Perhaps get ideas for themselves about how Tempo can be of more use to them.

**What would you like to be added**:

This could be links out to where we have existing documentation, and perhaps fill in gaps where users start to ask additional questions.  It could also include future work or updates to existing efforts where we do neat things user might get excited about.

This could include links out to https://play.grafana.org/ the traces panel, APM, node graph, service graph (and maybe how those differ).

**Who is this feature for?**

Comunity, but also those who give community support in slack so easy material to reference for those who are trying to make use of Tempo, and troubleshoot/understand a system under tracing instrumentation.

**Comment by GrafanaWriter** [comment]
@moxious - is this something your team might be interested in?

**Comment by moxious** [comment]
@GrafanaWriter yes, but perhaps not yet.  We have some tempo content on the roadmap as part of the learning journey but this seems to go a piece beyond into integrations.  Will ask this to be considered in the context of that roadmap

**Comment by knylander-grafana** [comment]
@moxious Curious for a status update on this issue. Thank you.

**Comment by moxious** [comment]
No status update.  Unfortunately tempo isn't within our scope at this time

**Comment by knylander-grafana** [comment]
Okay. I'll keep an eye on this. It's a good topic. I'm working on restructuring the tracing content for Tempo. I'll list this as a related issue.
